app/scripts/controllers/exportImport.js

```diff
@@ -126,6 +126,12 @@ angular.module('openhimConsoleApp')
       var exportData = angular.copy( $scope.selectedExports );
       var textFile = null;
 
+      // OHM-382: warn when exporting key
+      Alerting.AlertReset();
+      if(exportData !== undefined && exportData.Keystore !== undefined && exportData.Keystore.length > 0) {
```

Comment:
I would do this:
if(exportData && exportData.Keystore && exportData.Keystore.length > 0)